### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
There were many typos and many false positives - those were reverted.
Tool used: https://github.com/crate-ci/typos

### CODE REVIEW

File: packages/orama/tests/tokenizer.test.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ t.test('Tokenizer', async t => {
     const O1 = tokenizer.tokenize(I1, 'english')
     const O2 = tokenizer.tokenize(I2, 'english')
 
-    t.strictSame(O1, ['thi', 'is', 'a', 'test', 'with', 'test', 'duplic'])
+    t.strictSame(O1, ['this', 'is', 'a', 'test', 'with', 'test', 'duplic'])
```
Comment (szepeviktor) [inline packages/orama/tests/tokenizer.test.ts:71]: I am not sure here. Please add a commit to this PR when necessary.

Comment (micheleriva) [reply packages/orama/tests/tokenizer.test.ts:71]: Hi! Yes, this is meant to be truncated due to stemming. Can you please revert this? Everything else looks good to me. Thanks!

Comment (szepeviktor) [reply packages/orama/tests/tokenizer.test.ts:71]: I see! The computer thinks "this" is the plural of "thi" 🙃

Comment (szepeviktor) [reply packages/orama/tests/tokenizer.test.ts]: Done 🍏

**Review by micheleriva — APPROVED** [review]
LGTM, thanks!

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**